node/src/blockchain/blockchain_bridge.rs

```diff
@@ -1467,29 +1665,33 @@ mod tests {
                 context_id: 4321,
             }),
         };
-        let before = SystemTime::now();
+        // let before = SystemTime::now();
 
         let _ = addr.try_send(retrieve_transactions).unwrap();
 
         System::current().stop();
         system.run();
-        let after = SystemTime::now();
-        let accountant_received_payment = accountant_recording_arc.lock().unwrap();
-        let received_payments = accountant_received_payment.get_record::<ReceivedPayments>(0);
-        check_timestamp(before, received_payments.timestamp, after);
-        assert_eq!(
-            received_payments,
-            &ReceivedPayments {
-                timestamp: received_payments.timestamp,
-                payments: vec![],
-                new_start_block: 7,
-                response_skeleton_opt: Some(ResponseSkeleton {
-                    client_id: 1234,
-                    context_id: 4321
-                }),
-            }
-        );
-        TestLogHandler::new()
+        // let after = SystemTime::now();
+        let set_start_block_params = set_start_block_params_arc.lock().unwrap();
+        assert_eq!(*set_start_block_params, vec![Some(7)]);
+        // let accountant_received_payment = accountant_recording_arc.lock().unwrap();
+        // let received_payments = accountant_received_payment.get_record::<ReceivedPayments>(0);
+        // check_timestamp(before, received_payments.timestamp, after);
+        // assert_eq!(
+        //     received_payments,
+        //     &ReceivedPayments {
+        //         timestamp: received_payments.timestamp,
+        //         payments: vec![],
+        //         new_start_block: 7,
+        //         response_skeleton_opt: Some(ResponseSkeleton {
+        //             client_id: 1234,
+        //             context_id: 4321
+        //         }),
+        //     }
+        // );
+        let test_log_handler = TestLogHandler::new();
+        test_log_handler.exists_log_containing("DEBUG: BlockchainBridge: Write new start block: 7");
+        test_log_handler
```

Comment:
@bertllll Depending on your response, I will either uncomment the code in this test to make sure the `ReceivedPayments` message is always sent even if the transactions are empty so the `start_block_nbr` is updated in `accountant/scanner/mod`. The alternative is to not send the message when there are no transactions and update the `start_block_nbr` directly in `blockchain_bridge` and save the message sending overhead.